src/ws-header/ws-header.js

```diff
@@ -287,9 +287,16 @@ export class WSHeader extends Component {
                   ref={element => { this.menuItems[index] = element; }}
                   className={(link.isCurrent) ? 'is-current' : null}
                 >
-                  <a href={link.href} onClick={event => { if (link.onClick) link.onClick(event); }}>
-                    {link.label}
-                  </a>
+                {
+                  (
+                    link.$$typeof ? link :
```

Comment:
In general it's ok but pls keep the formatting as it was.
```
{link.$$typeof ?
  link
:
  <a href={link.href}>
    {link.label}
  </a>
}

Reply:
If you would add this in a standalone pr we could merge it

Reply:
👍